pkg/operator/api/userconfig/transformed_columns.go

```diff
@@ -19,8 +19,7 @@ package userconfig
 import (
 	"sort"
 
-	cr "github.com/cortexlabs/cortex/pkg/lib/configreader"
```

Comment:
There are probably a lot of other instances of `cr "github.com/cortexlabs/cortex/pkg/lib/configreader"`, do you want to switch them all? It can be a separate PR after this one